src/routes/dbInit/import.ts

```diff
@@ -0,0 +1,553 @@
+// tslint:disable
```

Comment:
This is... not okay. You're going to need to re-enable TSLint for this file and conform to the configured ruleset. It's going to take a bit of effort, but I'm willing to sit down with you and walk you through the refactor.

Reply:
Agreed, I would help as well. This PR needs to be rebased again because it does not contain the [models folder from dev](https://github.com/osu-cass/AP-CSE-API/tree/dev/src/models) and those types would help.